data_checker/README.md

```diff
@@ -0,0 +1,40 @@
+# Check of correct import of resource policies
+### We check ONLY if ANONYMOUS group has same READ access to ITEMS in Dspace5 and Dspace7 after success data import! We don't check other resource policies.
+
+## How to use it:
+1. Update `const.py`
+   - `user = "<ADMIN_NAME>"`
+   - `password = "<ADMIN_PASSWORD>"`
+
+   - `# http or https`
+   - `use_ssl = False`
+   - `host = "<YOUR_SERVER>" e.g., localhost`
+   - `# host = "dev-5.pc"`
+   - `fe_port = "<YOUR_FE_PORT>"`
+   - `# fe_port = ":4000"`
+   - `be_port = "<YOUR_BE_PORT>"`
+   - `# be_port = ":8080"`
+   - `be_location = "/server/"`
+   
+   - `authentication = False` - we test anonymous access

```

Comment:
Use comments like this inetead (relates to comment above)

Reply:
you wanna use own const file for this checker, so is setting of this line necessary here?

Reply:
well but you did not use it (which is ok).

This comment here is just that this is a good example.. of what comments should look like in readme.. as opposed to what I mentioned in comment above

Reply:
I thought that it is adding by yourself

Reply:
no, I was not adding it. I will not add it. If you think it is a good idea, add it. If not, then not. It was just a suggestion.

Reply:
ok, I think it is little bit complicated with it... I am going to use attribute in general const and add it to readme